driver/conv_driver.hpp

```diff
@@ -440,6 +440,23 @@ int ConvDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
     if(inflags.GetValueStr("out_layout").empty())
         inflags.SetValue("out_layout", default_layout);
 
+    std::string inLayoutStr  = inflags.GetValueStr("in_layout");
+    std::string  filLayoutStr = inflags.GetValueStr("fil_layout");
+    std::string outLayoutStr  = inflags.GetValueStr("out_layout");
+ 
+    std::transform(inLayoutStr.begin(), inLayoutStr.end(),inLayoutStr.begin(), ::toupper);
+    inflags.SetValue("in_layout", inLayoutStr);
+ 
+    std::transform(filLayoutStr.begin(), filLayoutStr.end(),filLayoutStr.begin(), ::toupper);
+    inflags.SetValue("fil_layout", filLayoutStr);
+   
+    std::transform(outLayoutStr.begin(), outLayoutStr.end(),outLayoutStr.begin(), ::toupper);
+    inflags.SetValue("out_layout", outLayoutStr);
+ 
+    inflags.SetValue("in_layout", inLayoutStr);
+    inflags.SetValue("fil_layout", filLayoutStr);
+    inflags.SetValue("out_layout", outLayoutStr);
```

Comment:
This is not necessary.

Reply:
This is regarding the parameter passing as an argument to MIOpen Driver via command line. 
 Flag names are 
a. in_layout
b. out_layout
c. fill_layout 

Currently above parameter are case sensitive and MIOpen driver is throwing an exceptional error  as a incorrect value.  To handle this exception,  we are making the parameter values as a it as case insensitive.

In the following example
MIOpenDriver convfp16 -n 128 -c 512 -H 28 -W 28 -k 128 -y 1 -x 1 -p 0 -q 0 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -F 1 -t 1 --in_layout nhwc --out_layout nhwc --fil_layout nhwc -> flag parameter values provided in lower case , system considered it as wrong value.  Error message was
'miopen::Exception'
what(): /root/dMIOpen/src/include/miopen/tensor_layout.hpp:62: mismatched layout string, unexpect char: N. 

Now, input/output/fill layout parameter values are no more case sensitive. If user provides lowercase values, it is handled as valid type.

Reply:
changes have been incorporated , redesigned to incorporate layout flags. New flag introduced in the library to check the case insensitive scenario.